R/plot_price.R

```diff
@@ -0,0 +1,14 @@
+#' Plot the price of cryptocurrency
+#'
+#' @param price_df data frame with time and price data.
+#' @param name character name of the cryptocurrency of interest
+#' @param window vector of two dates in character 
+#'
+#' @return ggplot object
+#' @export
+#'
+#' @examples
+#' plot_price(price_df, "BTCBitcoin", ["2021-01-01", "2021-12-31"])
+plot_price <- function(price_df, name, window){
```

Comment:
Should we remove the incomplete function below examples?

Reply:
The M2 requirements state, "Specifically, **set-up empty functions (with appropriate function names)** containing no code in the files where you will eventually write code." 

So I added it. 

> Should we remove the incomplete function below examples?